src/webgpu/api/operation/rendering/draw.spec.ts

```diff
@@ -57,7 +57,11 @@ Params:
       .expand('index_buffer_offset', p => (p.indexed ? ([0, 16] as const) : [undefined]))
       .expand('base_vertex', p => (p.indexed ? ([0, 9] as const) : [undefined]))
   )
-  .fn(t => {
+  .fn(async t => {
+    if (t.params.first_instance > 0 && t.params.indirect) {
+      await t.selectDeviceOrSkipTestCase('indirect-first-instance');
```

Comment:
```suggestion
      // TODO: 'as' cast because types don't have this feature name yet
      await t.selectDeviceOrSkipTestCase('indirect-first-instance' as GPUFeatureName);
```